tools/strip_nibabel

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Convert NiBabel objects in PyMVPA datasets attributes into a portable form
```

Comment:
how do we know that it works? ;-)
and not for me to tell you tales about "one time helpers" getting their own lives ;)

Let me finish this one up by
0. addressing sorted dict issue in comparison
1. push those test datasets somewhere accessible (I am thinking about having a git/annex clone under pymvpa.org/_files)
2. RF this one into a PyMVPA command with some minimal tests

Deal?